### ISSUE TITLE
Why no function for "Merge deletions only" or "Merge additions only" if you colorize those?

### ISSUE THREAD

**Issue by AjvarXX** [wickedest/Mergely#62]
By merge file it will merge both deleted lines and added lines. It would be twice more functional if it would also let merge deletions/additions only. Otherwise I had to click manually on red color only.

**Comment by wickedest** [comment]
*Merge file* does not actually merge - it copies one side to the other (I
should clarify that in the UI).  I can see how merging additions/deletions
only would be a good feature.

**Comment by AjvarXX** [comment]
Thank yo for considering the idea, glad I helped. Have a nice day.

**Comment by AjvarXX** [comment]
Yes, real life and more important projects can take it all, I understand. Mergely is free so more like a hobby afterall.
